### ISSUE TITLE
Please have a look at my pull request. Battery voltage and (dis)charge current are are displayed.

### ISSUE THREAD

**Issue by F4EWI** [lora-aprs/LoRa_APRS_Tracker#7]
Please have a look at my pull request. Battery voltage and (dis)charge current are are displayed.
Telemetry is still to-do.

_Originally posted by @raufaser in https://github.com/lora-aprs/LoRa_APRS_Tracker/issues/5#issuecomment-722700742_

I will test this this weekend

**Comment by peterus** [comment]
Please report when you have tested and if the changes are okay for you, thanks!

**Comment by F4EWI** [comment]
Yes for the tracker the voltage as well as the current is displayed well Thank you
![image](https://user-images.githubusercontent.com/74018524/98412647-9bfac780-2078-11eb-84af-d6330b95ea58.png)
But I can't see or I don't know how to see the telemetry information?
Maybe this is not available at the moment?
RSSI, SNR, Voltage and currant would be very interesting information to have in telemetry

**Comment by peterus** [comment]
telemetry is still not implemented, i will have to do this in the next days.
Voltage and current is possible. RSSI and SNR will be part of an update in the iGate firmware.

**Comment by peterus** [comment]
there is an update in #16 where we will send the battery voltage and current flow via plain text in the message, still no telemetry for charts etc.
